README.md

```diff
@@ -0,0 +1,38 @@
+[![Build Status](https://dev.azure.com/immunant/rexpat/_apis/build/status/immunant.rexpat?branchName=master)](https://dev.azure.com/immunant/rexpat/_build/latest?definitionId=2&branchName=master)
+
+# Rexpat: a libexpat compatible Rust crate
```

Comment:
It would be nice to have a bit here about the current status of the crate. Something like:

> This project is a work-in-progress conversion from unsafe Rust transpiled directly from libexpat into safe, idiomatic Rust code. Do not use this in production (yet!), but help refactoring and rewriting is always welcome.